src/MatrixClient.ts

```diff
@@ -677,6 +679,23 @@ export class MatrixClient extends EventEmitter {
         });
     }
 
+        /**
+     * Gets the joined members in a room. The client must be in the room to make this request.
```

Comment:
It gets any membership, not just joined. Also, the client having to be in the room is implicit or potentially irrelevant in the future.
```suggestion
     * Gets the members in a room. 
```